python/paddle/fluid/dygraph/jit.py

```diff
@@ -163,37 +185,19 @@ def func(x):
 
     """
 
-    def __impl__(*args, **kwargs):
-        program_translator = ProgramTranslator()
-        if not program_translator.enable_declarative:
-            warnings.warn(
-                "The decorator 'declarative' doesn't work when setting ProgramTranslator.enable=False. "
-                "We will just return dygraph output.")
-            return dygraph_func(*args, **kwargs)
-        try:
-            return program_translator.get_output(dygraph_func, *args, **kwargs)
-        except Exception as e:
-            error_data = getattr(e, ERROR_DATA, None)
-            if error_data:
-                new_exception = error_data.create_exception()
-                if six.PY3:
-                    # NOTE(liym27):
-                    # 1. Why `raise new_exception from None`?
-                    #   In Python 3, by default, an new exception is raised with trace information of the caught exception.
-                    #   This only raises new_exception and hides unwanted implementation details from tracebacks of the
-                    #   caught exception.
-                    # 2. Use exec to bypass syntax error checking in Python 2.
-
-                    six.exec_("raise new_exception from None")
-                else:
-                    raise new_exception
-            else:
-                raise
+    def _decorator_(python_func):
+        _, python_func = unwrap_decorators(python_func)
+        return make_decorator(
```

Comment:
This function doesn't look like "make" something, it copies something from original_func to decorator_func. Can you move the creation of StaticLayer into make_decorator to match your naming?

In addition, could you rename _decorator_ and make_decorator? They don't contain much information and I didn't understand their goal.

Reply:
Thanks, I have renamed them and added more comments.